File: python_programming/notes/Practices/old_enough.py
#EH 6th Old Enough 

age=int(input("Your age (in years): "))

if age >200:
    print("Wow, I don't know how you're still alive to type this! (But you can also vote. Congratulations!)")
elif age >=18:
    print("You are old enough to vote!")
elif age >=16:
    print("You are old enough to drive!")
elif age >=15:
    print("You are old enough to get a learners permit!")
elif age >=4:
    print("You are old enough to go to school!")
else:
    print("Wait until you are a little older!")

File: c_programming/practices/old_enough.c
// EH 6th Old Enough

#include <stdio.h>

int main(void){
    int age;
    printf("Enter your age (in years): ");
    scanf("%d", &age);

    if(age >=200){
    printf("Wow, I don't know how you're still alive to type this! (But you can also vote. Congratulations!)");
    }else if(age >=18){
    printf("You are old enough to vote!");
    }else if(age >=16){
    printf("You are old enough to drive!");
    }else if(age >=15){
    printf("You are old enough to get a learners permit!");
    }else if(age >=4){
    printf("You are old enough to go to school!");
    }else if(age < 0){
    printf("Wait until you're born sweetie.");
    }else{
    printf("You have your whole life ahead of you!");
    }

    return 0;
}